src/indigo/less/buttons.less

```diff
@@ -1,17 +1,27 @@
-.btn-link {
-  color: @text-color;
-}
+.btn {
+  .button-size(@padding-base * 3; @padding-base * 4; 11px; 14px; @border-radius);
+  font-weight: 500;
+  text-transform: uppercase;
+  letter-spacing: 1.09091px;
+
+  .btn-link {
```

Comment:
@natocTo tuna nie je typo? Naozaj to chces `.btn` a v nom bude `.btn-link`?

Reply:
Jop chyba, jdu to upravit.